resources/demoshop.md

```diff
@@ -17,9 +17,16 @@ sidebar:
 {% include alert.html type="warning"
                       icon="warning"
                       header="Site under development"
-                      body="The Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
+                      body="The Developer Portal is under construction and 
+                      should not be used to integrate against Swedbank Pay's 
+                      APIs yet." %}
 
 
 {% include jumbotron.html body=
 "We are working on a brand new demoshop for you!
-In the meantime, knock yourself out with our existing [demoshop](https://ecom.externalintegration.payex.com/pspdemoshop)" %}
+In the meantime, knock yourself out with our existing 
+[demoshop](https://ecom.externalintegration.payex.com/pspdemoshop)" %}
+
+{% include iterator.html prev_href="test-data" prev_title="Back: Test Data" 
```

Comment:
I'm not sure the iterator makes sense in the resource section, as there's no clear chronological order of the pages.

Reply:
Agreed. It makes sense that it doesn't make sense.
I'll remove them.